embedded/EscapeROOM/radio_morse_panel/radio_morse_panel.c

```diff
@@ -22,139 +48,84 @@ void unit()
 	}
 }
 
-void dot()
+void PlayDot()
 {
 	carrier = CARRIER_ON;
-	unit();
+	PlayUnit();
 	carrier = CARRIER_OFF;
 }
 
-void dash()
+void PlayDash()
 {
 	carrier = CARRIER_ON;
-	unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit();
 	carrier = CARRIER_OFF;
 }
 
-void short_space()
+void PlayShortSpace()
 {
-	unit();
+	PlayUnit();
 }
 
-void letter_space()
+void PlayLetterSpace()
 {
-	unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit();
 }
 
-void word_space()
+void PlayWordSpace()
 {
-	unit(); unit(); unit(); unit(); unit(); unit(); unit();
+	PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit(); PlayUnit();
```

Comment:
When you rename functions, then try to fo this in a single commit where the only change is function names.
Do name changes first and after that update algorithm or functionality. Don't mix - this is recipe for disaster.